manifests/k/KINGSGROUP/KOA/1.0/KINGSGROUP.KOA.locale.en-US.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: KINGSGROUP.KOA
+PackageVersion: 1.0
+PackageLocale: en-US
+Publisher: KINGS GROUP
+PackageName: King of Avalon
+License: Proprietary
+ShortDescription: King of Avalon is set in a fantasy medieval England bringing back the legend of King Arthur and the Knights of the Round Table.

```

```suggestion
ShortDescription: King of Avalon is set in a fantasy medieval England bringing back the legend of King Arthur and the Knights of the Round Table.
Moniker: koa
```
Abbreviated names like `KOA` instead of `KingOfAvalon` are best for monikers, as it will satisfy an exact match more easily